non-GPL/EventsTrace/EventsTrace.c

```diff
@@ -149,7 +150,38 @@ static void out_int(const char *name, const long value)
 
 static void out_string(const char *name, const char *value)
 {
-    printf("\"%s\":\"%s\"", name, value);
+    printf("\"%s\":\"", name);
+    for (size_t i = 0; i < strlen(value); i++) {
+        char c = value[i];
+        switch (c) {
+        case '\n':
+            printf("\\n");
+            break;
+        case '\r':
+            printf("\\r");
+            break;
+        case '\\':
+            printf("\\\\");
+            break;
+        case '"':
+            printf("\"");
+            break;
+        case '\t':
+            printf("\\t");
+            break;
+        case '\b':
+            printf("\\b");
+        default:
+            printf("%c", c);
+            break;
```

Comment:
should this be:
```suggestion
            break;
        default:
```
?

Reply:
🙏 tyvm, I was far too tired Thursday evening 😛